shielder/contract/drink_tests/utils/shielder.rs

```diff
@@ -58,19 +57,17 @@ pub fn create_shielder_account(
 
     let h_note_new = Note::new(id, trapdoor, nullifier, acc.hash()).hash();
 
-    session.call_with_address(
+    let leaf_id_res: Result<u32, ()> = session.call_with_address(
```

Comment:
you should use true error type - in case of regression, we will get decoding error instead of true variant, which might be misleading

Reply:
using it now